deps.toml

```diff
@@ -95,7 +95,7 @@ platform-testcontainers-jdbc = { module = "org.testcontainers:jdbc", version.ref
 platform-testcontainers-postgresql = { module = "org.testcontainers:postgresql", version.ref = "platform-testcontainers" }
 postgresql = { module = "org.postgresql:postgresql", version.ref = "postgresql" }
 quartz-scheduler = { module = "org.quartz-scheduler:quartz", version = "2.3.2" }
-s3 = { module = "software.amazon.awssdk:s3", version = "2.16.84" }
+s3 = { module = "software.amazon.awssdk:s3", version = "2.19.4" }
```

Comment:
I guess that we will see with the acceptance test but we might change how we call this lib. THis was likely the cause of the issue previously.

Reply:
The unknown host exception is the same error I was seeing when I was testing with `3.8.0`.
To clarify what I was saying earlier, I think the issues I saw when I tested`3.7.5` may have been fixed by the changes I merged yesterday.